docs/reverse_proxy.md

```diff
@@ -66,6 +67,40 @@ server {
 **NOTE**: Do not add a path after the port in `proxy_pass`, otherwise nginx will
 canonicalise/normalise the URI.
 
+### Kong
+
+```yaml
+_format_version: "2.1"
+_transform: true
+
+# Synapse
+services:
+- name: synapse-service
+  url: http://127.0.0.1:8008
+  routes:
+  - name: synapse-client-route
+    paths:
+    - /_matrix
+    - /_synapse/client
```

Comment:
This also covers `/_matrix/federation/...` routes (server<->server communication), so `synapse-client-route` may be slightly misleading. Perhaps `synapse-public-route` instead?